help/destinations/catalog/advertising/pubmatic.md

```diff
@@ -1,20 +1,29 @@
 ---
 title: PubMatic Connect
 description: PubMatic maximizes customer value by delivering the programmatic digital marketing supply chain of the future. PubMatic Connect combines platform technology and dedicated service to enhance how inventory and data are packaged and transacted.
-last-substantial-update: 2023-12-14
+last-substantial-update: 2025-02-12
 exl-id: 21e07d2c-9a6a-4cfa-a4b8-7ca48613956c
 ---
+
 # PubMatic Connect destination {#pubmatic-connect}
 
 ## Overview {#overview}
 
 Use [!DNL PubMatic Connect] to maximize customer value by delivering the programmatic digital marketing supply chain of the future. [!DNL PubMatic Connect] combines platform technology and dedicated service to enhance how inventory and data are packaged and transacted.
 
-Use this destination to send audience data to the [!DNL PubMatic Connect] platform.
+There are two destinations available that let you send audience data to the PubMatic Connect platform. They differ slighty in their functionality:
+
+1. PubMatic Connect
+
+   During the initial activation, this destination will automatically register the segments in the PubMatic platform and use the internal Adobe Experience Platform ID for mapping.
```

Comment:
we usually talk about audiences instead of segments in Experience Platform.

Reply:
Changed!